src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -434,6 +438,44 @@ private IOperation CreateBoundCallOperation(BoundCall boundCall)
             return new CSharpLazyInvocationOperation(this, boundCall, targetMethod.GetPublicSymbol(), isVirtual, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
+        private IOperation CreateBoundFunctionPointerInvocationOperation(BoundFunctionPointerInvocation boundFunctionPointerInvocation)
+        {
+            if (boundFunctionPointerInvocation.HasErrors)
```

Comment:
>.HasErrors [](start = 46, length = 10)

This is a very imprecise test. Is it possible to make it more specific, check ```ResultKind```, for example?

Reply:
Yeah, I think we could. I hadn't added `ResultKind` yet when I implemented this part, I think it's probably a better fit.

---
In reply to: [428786732](https://github.com/dotnet/roslyn/pull/44436#discussion_r428786732) [](ancestors = 428786732)